tests/e2e/util.go

```diff
@@ -1676,23 +1678,65 @@ func VsanObjIndentities(ctx context.Context, vs *vSphere) []string {
 		framework.Logf("Default cluster is choosen for test")
 	}
 	clusterComputeResource, vsanHealthClient, err := getClusterName(ctx, vs)
-	framework.Logf("clusterComputeResource %v", clusterComputeResource)
 	gomega.Expect(err).NotTo(gomega.HaveOccurred())
 
 	for _, cluster := range clusterComputeResource {
-		framework.Logf("cluster name %v", cluster.Name())
+		framework.Logf("clusterComputeResource %v", clusterComputeResource)
 		if strings.Contains(cluster.Name(), computeCluster) {
 			clusterConfig, err := vsanHealthClient.VsanQueryObjectIdentities(ctx, cluster.Reference())
+			framework.Logf("clusterconfig %v", clusterConfig)
 			gomega.Expect(err).NotTo(gomega.HaveOccurred())
-			vsanObjUUID = clusterConfig.Health.ObjectHealthDetail[0].ObjUuids
-			break
+			for index := range clusterConfig.Identities {
+				if strings.Contains(clusterConfig.Identities[index].Description, pvName) {
+					vsanObjUUID = clusterConfig.Identities[index].Uuid
+					framework.Logf("vsanObjUUID is %v", vsanObjUUID)
+					break
+				}
+			}
 		}
 	}
+	gomega.Expect(vsanObjUUID).NotTo(gomega.BeEmpty())
 	return vsanObjUUID
 }
 
+func findIP(input string) string {
+	numBlock := "(25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])"
+	regexPattern := numBlock + "\\." + numBlock + "\\." + numBlock + "\\." + numBlock
+
+	regEx := regexp.MustCompile(regexPattern)
+	return regEx.FindString(input)
+}
+
+func checkAllHostStatus(ctx context.Context, vs *vSphere) {
+	computeCluster := os.Getenv("CLUSTER_NAME")
```

Comment:
If we are using a different testbed which has a different `CLUSTER_NAME` then this ENV is helpful, If we are not passing `CLUSTER_NAME` then it will take the default cluster name.
This we have agreed in the last code review.

Reply:
That is fine, what I asked was different,
take this line as an example from e2e_common.go,
`envStoragePolicyNameForSharedDatastores    = "STORAGE_POLICY_FOR_SHARED_DATASTORES"`

here the env param is `"STORAGE_POLICY_FOR_SHARED_DATASTORES"` just like `"CLUSTER_NAME"` here